src/plugins/wb-delete-pii/wb-delete-pii.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @title WET-BOEW wb-delete-pii
+ * @overview This plugin delete Personal Identifiable Information (PII) before form submit
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @polmih
+ **/
+( function( $, wb ) {
+"use strict";
+
+var $document = wb.doc,
+	componentName = "wb-delete-pii",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector );
+
+		if ( elm ) {
+			var $elm = $( elm );
+
+			elm.addEventListener( "submit", function() {
+
+				// identify form elements that was assigned to be scrubbed
+				$( "[data-wb-pii]", $elm ).each( function() {
+
+					var $elmInput = $( this ),
+						elmInputVal = $elmInput.val(),
+						inpValLenght = elmInputVal.length;
+
+					if ( $elmInput.attr( "data-wb-pii" ) === "true" ) {
+
+						// if the helper find any PII values clean the PII and assign new value to the form element with the output string
+						$elmInput.val( wb.findPotentialPII( elmInputVal, true ) === false ? elmInputVal : wb.findPotentialPII( elmInputVal, true ) );
+
+						// if any form element value was scrubbed create and submit an hidden input with the name="scrubb" and value="1"
+						if ( $elmInput.val().length !== inpValLenght ) {
+							var input = elm.querySelector( "input[name=\"scrubb\"]" );
+							if ( !input ) {
+								input = document.createElement( "input" );
+								input.setAttribute( "type", "hidden" );
+								input.setAttribute( "name", "scrubb" );
+								input.setAttribute( "value", "1" );
```

Comment:
It will be easier to use jQuery instead.
```suggestion
$input = $( "<input type='hidden' name='" + name + "' />" );
$input.val( 1 );
```